### ISSUE TITLE
RTD build fails / no 0.6.2 docs online

### ISSUE THREAD

**Issue by orbeckst** [Becksteinlab/GromacsWrapper#118]
I hate debugging RTD failures... 

No idea why https://readthedocs.org/projects/gromacswrapper/builds/5188013/ fails but locally it builds just fine.

Anyone ( @dotsdl , @jandom , @ianmkenney ) can shed some light on this?

Note:
```
python setup.py build_sphinx upload_docs
```
builds but then
```
running upload_docs
Upload_docs command is deprecated. Use RTD instead.
Submitting documentation to https://pypi.python.org/pypi/
Upload failed (503): first byte timeout
```
Note that it says *Upload_docs command is deprecated. Use RTD instead.* so maybe that's what is supposed to work.

For completeness: I also tried manually uploading docs to PyPi as a zip file but ran into an error like pypa/pypi-legacy#266 (and specifically pypa/pypi-legacy#454)

```
Error 503 first byte timeout

first byte timeout
Guru Mediation:

Details: cache-sea1047-SEA 1490291444 1413371420

Varnish cache server
```

**Comment by orbeckst** [comment]
Apparentlt, pythonhosted is in read-only mode https://github.com/MDAnalysis/mdanalysis/issues/664#issuecomment-293033814 – so we need to sort out RTD or use gh-pages.

**Comment by dotsdl** [comment]
In f315770 I've added the sphinx conda package to the environment we install. This ensures we get the latest version, since RTD often runs at least a release behind if not more. Looks like builds are working again.

**Comment by orbeckst** [comment]
Awesome, thanks!

--
Oliver Beckstein
email: orbeckst@gmail.com

**Comment by orbeckst** [comment]
Thanks again. I backported (cherry-picked) your fix onto master and now the 0.6.2 docs are also properly available again.
